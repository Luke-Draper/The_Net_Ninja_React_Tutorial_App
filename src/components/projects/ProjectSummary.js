import React, { Component } from "react";

class ProjectSummary extends Component {
	render() {
		return (
			<div className="project-summary">
				<h2>ProjectSummary</h2>
			</div>
		);
	}
}

export default ProjectSummary;
