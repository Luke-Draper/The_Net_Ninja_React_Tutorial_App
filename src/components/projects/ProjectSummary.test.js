import React from "react";
import ProjectSummary from "./ProjectSummary";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("ProjectSummary renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<ProjectSummary />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
