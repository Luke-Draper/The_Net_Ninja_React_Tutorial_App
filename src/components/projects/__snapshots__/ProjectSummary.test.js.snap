// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectSummary renders without crashing 1`] = `
<div
  className="card z-depth-0 project-summary"
>
  <div
    className="card-content grey-text text-darken-3"
  >
    <span
      className="card-title"
    >
      Project Title
    </span>
    <p>
      Posted by Luke Draper
    </p>
    <p
      className="grey-text"
    >
      3rd September, 2018
    </p>
  </div>
</div>
`;
